tests/storage.spec.ts

```diff
@@ -153,3 +159,39 @@ test("storage volume of type block", async ({ page }) => {
 
   await deleteVolume(page, volume);
 });
+
+test("duplicate custom storage volume", async ({ page }) => {
+  const newProjectName = randomProjectName();
+  const newPoolName = randomPoolName();
+  await createPool(page, newPoolName);
+  await createProject(page, newProjectName);
+
+  // duplicate volume in the same project and same pool
+  let newVolumeName = await duplicateVolume(page, volume);
```

Comment:
```suggestion
  let duplicateVolume = await duplicateVolume(page, volume);
```

Reply:
the variable name and function name will clash, so I renamed the function name to `duplicateStorageVolume`.